ext/pdo_pgsql/pgsql_driver.c

```diff
@@ -15,6 +15,7 @@
   | Authors: Edin Kadribasic <edink@emini.dk>                            |
   |          Ilia Alshanestsky <ilia@prohost.org>                        |
   |          Wez Furlong <wez@php.net>                                   |
+  |          Pablo Santiago Sánchez <phackwer@gmail.com>                 |
```

Comment:
hmm, we usually won't edit copyright fields in this case...

Reply:
Years after this discussion I've just realised that I've removed one of the biggest annoying barriers that made newbies to avoid PostgreSQL and use MySQL/MariaDB instead. I really think that my name should have been included in the authors list. It may seem a "dumb" feature, but that's the dumb feature that do note prevent new developers to use PG!

Anyway... just wanted to add this here.